CommandLine/testing/Platform/TestHarness-X11.cpp

```diff
@@ -99,6 +99,10 @@ class X11_TestHarness final: public TestHarness {
   void close_window() final {
     system(("wmctrl -i -c " + to_string(window_id)).c_str());
   }
+  void screen_save(std::string fPath) final {
+    sleep(3); // give the game a chance to render a frame first
```

Comment:
There's no need to do this now because I made `draw_test.cpp` call test harness `wait()` which basically accomplishes this in a more straight-forward way. Then it's clear like the `window_test.cpp` and I was going to change #1291 to do the same.